features/support/cucumber_rails_helper.rb

```diff
@@ -50,8 +51,7 @@ def validate_rails_new_success(result)
 
   def clear_bundle_env_vars(dir)
     cd dir
-    delete_environment_variable 'RUBYOPT'
-    delete_environment_variable 'BUNDLE_BIN_PATH'
+    unset_bundler_env_vars
```

Comment:
Use the Aruba gem's support for unsetting Bundler's environment configuration. There were a few more variables that were tripping up GitHub Actions.